components/arm/arm.go

```diff
@@ -107,15 +108,19 @@ func CreateStatus(ctx context.Context, a Arm) (*pb.Status, error) {
 		return nil, err
 	}
 	model := a.ModelFrame()
-	endPosition, err := motionplan.ComputeOOBPosition(model, jointPositions)
-	if err != nil {
-		return nil, err
+	var endPosition *v1.Pose
+	if jointPositions != nil && model != nil {
+		endPose, err := motionplan.ComputeOOBPosition(model, jointPositions)
+		if err != nil {
+			return nil, err
+		}
+		endPosition = spatialmath.PoseToProtobuf(endPose)
 	}
 	isMoving, err := a.IsMoving(ctx)
 	if err != nil {
 		return nil, err
 	}
-	return &pb.Status{EndPosition: spatialmath.PoseToProtobuf(endPosition), JointPositions: jointPositions, IsMoving: isMoving}, nil
+	return &pb.Status{EndPosition: endPosition, JointPositions: jointPositions, IsMoving: isMoving}, nil
```

Comment:
@randhid  and I talked and think that we should be doing something like the pseudocode below 
```
		endPose, err := motionplan.ComputeOOBPosition(model, jointPositions)
		if err != nil {
			// append to the status
		}
```

Reply:
If the end position errors out then we should not add it to the status